### DISCUSSION TITLE
doris2.1.2-flink-cdc-connector无法开启服务端攒批功能

### DISCUSSION THREAD

**Discussion by paul1989889** [apache/doris#35000] [A - General / Q&A]
在最新的2.1.2版本中，我尝试使用了攒批的能力，我的写入场景是通过flink-cdc-connector进行同步, 在正常的stream-load模式下，header中添加group-commit:async-mode正常运行，result也显示使用了服务端攒批的能力，但我在flink中使用bacth-mode(flink的攒批能力)，result显示并非使用了服务端攒批能力，而是flink的BatchWriter进行了攒批，基于以上问题，我将flink的sink参数，sink.properties.group_commit 设置为async-mode，猜想是否能同时启用flink的攒批能力及服务端（BE）的攒批能力，结果得到以下错误:
Caused by: org.apache.flink.util.SerializedThrowable: org.apache.doris.flink.exception.DorisBatchLoadException: stream load error: [INTERNAL_ERROR]label and group_commit can't be set at the same time

    0#  doris::StreamLoadAction::_handle_group_commit(doris::HttpRequest*, std::shared_ptr<doris::StreamLoadContext>) at /home/zcp/repo_center/doris_release/doris/be/src/common/status.h:425
    1#  doris::StreamLoadAction::on_header(doris::HttpRequest*) at /var/local/ldb_toolchain/bin/../lib/gcc/x86_64-linux-gnu/11/../../../../include/c++/11/bits/shared_ptr_base.h:701
    2#  doris::EvHttpServer::on_header(evhttp_request*) at /home/zcp/repo_center/doris_release/doris/be/src/http/ev_http_server.cpp:255
    3#  ?
    4#  bufferevent_run_readcb_
    5#  ?
    6#  ?
    7#  ?
    8#  ?
    9#  std::_Function_handler<void (), doris::EvHttpServer::start()::$_0>::_M_invoke(std::_Any_data const&) at /var/local/ldb_toolchain/bin/../lib/gcc/x86_64-linux-gnu/11/../../../../include/c++/11/ext/atomicity.h:98
    10# doris::ThreadPool::dispatch_thread() at /home/zcp/repo_center/doris_release/doris/be/src/util/threadpool.cpp:0
    11# doris::thread::supervise_thread(void*) at /var/local/ldb_toolchain/bin/../usr/include/pthread.h:562
    12# start_thread
    13# __clone
, see more in null
    at org.apache.doris.flink.sink.batch.DorisBatchStreamLoad$LoadAsyncExecutor.load(DorisBatchStreamLoad.java:292)
    ... 4 common frames omitted
我是否可以理解为，在flink-cdc-connector开启batch-mode同步的过程中，还是使用的是客户端（flink）的攒批能力，而非服务端的攒批能力，而目前只有http-stream和load-stream以及JDBC方式支持服务端攒批能力?

**Comment by yagagagaga** [comment]
group_commit is not yet supported in flink-doris-connector.
